Packs/CommonScripts/Scripts/SSDeepSimilarity/SSDeepSimilarity_test.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+import demistomock as demisto  # noqa: F401
+
+CASE_ANCHOR_EMPTY = {'ssdeep_hash': '',
+                     'ssdeep_hashes_to_compare': '3:AXGBicFlIHBGcL6wCrFQEv:AXGH6xLsr2C'}
+CASE_HASH_LIST_EMPTY = {'ssdeep_hash': '3:AXGBicFlgVNhBGcL6wCrFQEv:AXGHsNhxLsr2C',
+                        'ssdeep_hashes_to_compare': ''}
+CASE_VALID_INPUT = {'ssdeep_hash': '3:AXGBicFlgVNhBGcL6wCrFQEv:AXGHsNhxLsr2C',
+                    'ssdeep_hashes_to_compare':
+                        '3:AXGBicFlIHBGcL6wCrFQEv:AXGH6xLsr2C, 3:AXGBicFlgVNhBGcL6wCrFQEv:AXGHsNhxLsr2C',
+                    'output_key': 'test'}
+
+INPUT_INVALID_CASES = [
+    (CASE_ANCHOR_EMPTY, 'Please provide an hash to compare to.'),
+    (CASE_HASH_LIST_EMPTY, 'Please provide at least one hash to compare with.'),
+
+]
+
+
+@pytest.mark.parametrize('case_inputs, expected_error_msg', INPUT_INVALID_CASES)
+def test_handle_inputs_fails_on_invalid_input(case_inputs, expected_error_msg):
```

Comment:
let's add a case for giving a SSDeep hash that is invalid(not valid chars inside/length/etc)